asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -206,43 +206,42 @@ xsd:date: *2001-10-26* -> HL7 DTM: *20011026*
+.R8120
+[sdpi_requirement#r8120,sdpi_req_level=shall,sdpi_max_occurrence=1]
+****
+If <<r8102>> is met, then a <<actor_somds_dec_gateway>> / <<actor_somds_acm_gateway>> shall export the patient's sex as OBX segment on the MDS level.
 
-. *Map the "sex for clinical use" in a separate OBX segment (only possible for the DEC Profile but not for ACM) and the Administrative Sex/Gender to the PID-8 Administrative Sex field*: the administrative sex/gender information is mapped to the correctly named PID-8 "Administrative Sex" field, and the sex for clinical use is provided in a separate OBX segment in the IHE DEC Profile conform export. The sex for clinical use information will be not available in the ACM Profile conform export. Pros: the information are mapped to the correct fields. Cons: this mapping might not be backward compatible to the existing IHE DEV/PCD profile actors (reporter/consumer). Information of the sex for clinical use may get lost in some profiles such as ACM.
+.Notes
+[%collapsible]
+====
+NOTE: The mapping for the patient's sex is defined in table <<ref_tbl_dec_obx_sex_mapping>>.
+====
+****
 
-*REVIEWER QUESTION*:  Please review the options listed above and provide comments on the mapping of this semantic content.
+.R8121
+[sdpi_requirement#r8121,sdpi_req_level=may,sdpi_max_occurrence=1]
+****
+If <<r8102>> is met and the patient's sex in the <<acronym_mdib>> is sourced from the PID-8 field in HL7 V2 ADT messages provided by the hospital ADT system, then a <<actor_somds_dec_gateway>> / <<actor_somds_acm_gateway>> may set the PID-8 field to the code for the administrative sex.
 
-|===
+.Notes
+[%collapsible]
+====
+NOTE: <<ref_tbl_pid8_mapping>> defines the mapping of the SDC patient's sex information to the data fields of the HL7 data type *IS* used in the PID-8 field.
+====
+****
 
 .R8107
 [sdpi_requirement#r8107,sdpi_req_level=shall,sdpi_max_occurrence=1]
 ****
-If <<r8102>> is met, then a <<actor_somds_dec_gateway>> / <<actor_somds_acm_gateway>> shall set the PID-8 field to the code for the administrative sex.
+If <<r8102>> is met and the <<actor_somds_v2_gateway>> is configured to export the patient's sex in the PID-8 field, then a <<actor_somds_dec_gateway>> / <<actor_somds_acm_gateway>> shall set the PID-8 field to the code for the administrative sex.
 
-However, in the clinical context of a point of care device the biological sex, or sex at birth is important for various algorithms.
+The organization has to consider the risk that the patient's sex set in the <<acronym_mdib>> and mapped to the PID-8 field does not lead to a misinterpretation of the sex concept on <<actor_somds_v2_gateway>> consumer side.

```

Comment:
Should be a shall requirement:
If the <actor> provides the HDO with means to configure the <actor> to export the patient’s sex in the PID-8 field, the <actor> SHALL require in the ACCOMPANYING INFORMATION that the HDO considers the risk of misinterpretation.

Reply:
Added requirement R8122